File: academic-system/src/main/java/org/example/academic/system/repository/PersistenceConfiguration.java
package org.example.academic.system.repository;

public class PersistenceConfiguration {

    private static PersistenceType currentType = PersistenceType.TXT;

    public static PersistenceType getCurrentType() {
        return currentType;
    }

    public static void setCurrentType(PersistenceType type) {
        currentType = type;
    }
}

File: academic-system-python/src/academic_system/repository/persistence_configuration.py
from academic_system.repository.persistence_type import PersistenceType


class PersistenceConfiguration:
    _current_type: PersistenceType = PersistenceType.TXT

    @classmethod
    def get_current_type(cls) -> PersistenceType:
        return cls._current_type

    @classmethod
    def set_current_type(cls, persistence_type: PersistenceType):
        cls._current_type = persistence_type